path_config.go

```diff
@@ -200,20 +183,19 @@ func (b *azureSecretBackend) pathConfigRead(ctx context.Context, req *logical.Re
 
 	resp := &logical.Response{
 		Data: map[string]interface{}{
-			"subscription_id":         config.SubscriptionID,
-			"tenant_id":               config.TenantID,
-			"environment":             config.Environment,
-			"client_id":               config.ClientID,
-			"use_microsoft_graph_api": config.UseMsGraphAPI,
-			"root_password_ttl":       int(config.RootPasswordTTL.Seconds()),
+			"subscription_id":   config.SubscriptionID,
+			"tenant_id":         config.TenantID,
+			"environment":       config.Environment,
+			"client_id":         config.ClientID,
+			"root_password_ttl": int(config.RootPasswordTTL.Seconds()),
```

Comment:
Similar to what @fairclothjm mentioned in a different comment, I think we can maintain the `use_microsoft_graph_api` field in the read response. Its value can be hard-coded to `true`, since that's the only value that it can take after this point.

I could also see the argument for removing it completely, but let's see if that makes sense for what we've communicated in terms of deprecation. I'll check on this to be sure 👍

Reply:
I'm starting to think it would be okay to remove the `use_microsoft_graph_api` field entirely for both writes and reads. Its purpose was to allow users to transition from AAD graph to MS graph during the deprecation phase. We've communicated the deprecation over the span of two releases (announcements went out in Vault 1.10 with the introduction of the field). It would serve no functional purpose after we've removed the AAD graph code. Let's see what others think.

Reply:
I'm good with removing `use_microsoft_graph_api` entirely for reads and writes.  👍 

My concern was that we would fail requests that provided the field. But I did some tests and it looks like we just ignore it. (This behavior in vault of ignoring unknown fields always catches me off guard.)

Reply:
Thanks for the input, @fairclothjm! Sounds good to me. (+1 on the ignoring unknown fields point).